tests/gdx-tests-android/src/com/badlogic/gdx/tests/android/AndroidTestStarter.java

```diff
@@ -38,6 +37,8 @@
 	public void onCreate (Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		GdxTests.tests.add(MatrixTest.class);
+		if (!GdxTests.tests.contains(APKExpansionTest.class))

```

Comment:
And the check is needed because I noticed that otherwise it can happen that it will be added twice to the list if you do a back and start GDX Tests again.